visidata/features/menu_guide.py

```diff
@@ -0,0 +1,29 @@
+from visidata import GuideSheet, vd
+
+class MenuGuide(GuideSheet):
+    guide_text = '''# The Menu System
+
+Located at the [:black on 68]very top of the screen[/], it contains a collection of commands organised within submenu trees.
+
+The menu can be navigated both as a standard, clickable GUI menu, and via keystrokes.
+
+To navigate via keystrokes, `Alt+H` will open the **Help** menu, and from there `↑ ↓ → ←` keys can be used to move between submenus.
+
+Additionally, each top-level menu name has a single letter underlined. `Alt+<underlined letter>` will open that menu. For example, `Alt+F` will open the **File** menu.
+
+`{vd.options.disp_menu_more}` indicates a submenu, which can be traversed with the `→` or a mouse-click. The leaf node of every menu tree is a command.
+
+When the cursor is on a command, the menu throws up its [:color_menu_help]helpbox[/]. This contains the description of the command, the keyboard shortcut if available, and its command longname. Clicking on the command in the menu, or pressing `Enter`, will execute the actual command.
```

Comment:
"throws up"?  How about "The menu shows the description of the current command, along with the command name and its keyboard shortcut (if available)."
- avoid future/conditional tenses; use imperative/infinitive instead: "Execute the current command by clicking on it or pressing `Enter`".